withings_api/common.py

```diff
@@ -49,6 +49,7 @@ def new_sleep_state(value: Optional[int]) -> SleepState:
 class MeasureGetMeasGroupAttrib(IntEnum):
     """Measure group attributions."""
 
+    UNKNOWN = -1
```

Comment:
Interesting. Are you seeing -1 values with data coming back from your account? This value is undocumented in the API spec.

Reply:
Yeah, Here's a sample: {'grpid': 250284004, 'attrib': -1, 'date': 1409730164  , 'created': 1409730234, 'category': 1, 'deviceid': '0842f39947b59c73cb3662b97dca949b88e2ffa2', 'measures': [{'value': 100, 'type': 9, 'unit': 0, 'algo': 0, 'fw': 0, 'fm': 0}, {'value': 131, 'type': 10, 'unit': 0, 'algo': 0, 'fw': 0,   'fm': 0}, {'value': 94, 'type': 11, 'unit': 0, 'algo': 0, 'fw': 0, 'fm': 0}], 'comment': None}